### ISSUE TITLE
Dependencies missing on generated HTML files

### ISSUE THREAD

**Issue by kbariotis** [parcel-bundler/parcel#440]
This is a 🐛 bug report.

### 🎛 Configuration (.babelrc, package.json, cli command)

**index.html**
```html
<!DOCTYPE html>
<html lang="en">
<head>
  <title>Document</title>
</head>
<body>
  <h1>Home</h1>

  <a href="./about.html">About</a>

  <script src="./index.js"/>
</body>
</html>
```

**about.html**
```html
<!DOCTYPE html>
<html lang="en">
<head>
  <title>Document</title>
</head>
<body>
  <h1>About</h1>

  <a href="./index.html">About</a>

  <script src="./index.js"/>
</body>
</html>
```
**index.js**
```js
require('./boot.scss')
```

### 🤔 Expected Behavior

Both generated `.html` files to have a `link` tag pointing to the generated `.css` file.

### 😯 Current Behavior

Only `about.html` contains a `link` tag.

### 🔦 Context

I am trying to implement a Pug asset plugin and stumbled upon this issue. At first I though it had to with the inheritance mode of Pug but then I tested with raw html and produced the same results. You can find my [code here](https://github.com/kbariotis/parcel-bootstrap-pug).

### 🌍 Your Environment

| Software         | Version(s) |
| ---------------- | ---------- |
| Parcel           | 1.3.1 |
| Node             | 8.9.3 |
| npm/Yarn         | 5.5.1 |
| Operating System | OSX High Sierra |

**Comment by devongovett** [comment]
This is potentially already fixed by #380 which is merged but not released to npm yet. Can you test using the code in master?

**Comment by devongovett** [comment]
Now released in v1.4.0. Please test and reopen the issue if you still have a problem.

**Comment by kbariotis** [comment]
Fixed indeed @devongovett. Thanks :)
